template.rb

```diff
@@ -121,6 +121,7 @@ def apply_template!
     # lighthouse matcher parts we need to run performance specs
     apply "variants/performance/template.rb"
     apply "variants/bullet/template.rb"
+    apply "variants/pundit/template.rb"
```

Comment:
I'm making pundit required. I don't think we have every created an app without it.